src/core/lib/security/credentials/tls/grpc_tls_crl_provider.cc

```diff
@@ -53,15 +53,22 @@ namespace grpc_core {
 namespace experimental {
 
 namespace {
-std::string IssuerFromCrl(X509_CRL* crl) {
+// TODO(gtcooke94) Move ssl_transport_security_utils to it's own BUILD target
+// and add this to it.
+absl::StatusOr<std::string> IssuerFromCrl(X509_CRL* crl) {
   if (crl == nullptr) {
     return "";
```

Comment:
Shouldn't this return a non-OK status as well?  Previously, we were failing if the returned string was empty.

Reply:
Oh yes, good catch, fixing it

Reply:
Was trying to get test coverage, and I think that, given the current codepaths it is not currently _actually_ possible to have `crl == nullptr` as an input to this function as its only caller does the same check (and this fn is just defined in the the anonymous namespace in this `.cc` file, so it's not defined elsewhere).